example/satosa/pyeudiw_backend.yaml

```
@@ -220,14 +220,19 @@ config:
             n: utqtxbs-jnK0cPsV7aRkkZKA9t4S-WSZa3nCZtYIKDpgLnR_qcpeF0diJZvKOqXmj2cXaKFUE-8uHKAHo7BL7T-Rj2x3vGESh7SG1pE0thDGlXj4yNsg0qNvCXtk703L2H3i1UXwx6nq1uFxD2EcOE4a6qDYBI16Zl71TUZktJwmOejoHl16CPWqDLGo9GUSk_MmHOV20m4wXWkB4qbvpWVY8H6b2a0rB1B1YPOs5ZLYarSYZgjDEg6DMtZ4NgiwZ-4N1aaLwyO-GLwt9Vf-NBKwoxeRyD3zWE2FXRFBbhKGksMrCGnFDsNl5JTlPjaM3kYyImE941ggcuc495m-Fw
             p: 2zmGXIMCEHPphw778YjVTar1eycih6fFSJ4I4bl1iq167GqO0PjlOx6CZ1-OdBTVU7HfrYRiUK_BnGRdPDn-DQghwwkB79ZdHWL14wXnpB5y-boHz_LxvjsEqXtuQYcIkidOGaMG68XNT1nM4F9a8UKFr5hHYT5_UIQSwsxlRQ0
             q: 2jMFt2iFrdaYabdXuB4QMboVjPvbLA-IVb6_0hSG_-EueGBvgcBxdFGIZaG6kqHqlB7qMsSzdptU0vn6IgmCZnX-Hlt6c5X7JB_q91PZMLTO01pbZ2Bk58GloalCHnw_mjPh0YPviH5jGoWM5RHyl_HDDMI-UeLkzP7ImxGizrM
-    # x509:
-      # module: pyeudiw.trust.handler.x509
-      # class: X509TrustModel
-      # config:
-        # trust_anchor_certificates:
-            # - "todo"
-        # trust_anchors_cn: # we might mix CN and SAN together
-            # - http://127.0.0.1:8000
+    x509:
+      module: pyeudiw.trust.handler.x509
+      class: X509TrustHandler
+      config:
+        client_id: *client_id
+        client_id_scheme: x509_san_dns
```

```suggestion
        client_id_scheme: x509_san_dns # this will be prepended in the client id scheme used in the request. 
```

the point is therefore the following:

enabling more than a single trust evaluation handler, thefore enabling more than a single client id ... supposing to support both x.509 and federation:

- the last trust handler client id (and client id scheme too, if present) should override the client id used in the main presentation request

nothing prevents to make a wallet able to guess or discover other trust evaluation mechanisms, if available.